### ISSUE TITLE
Paradox of 3 training schedules???

### ISSUE THREAD

**Issue by YJHMITWEB** [facebookresearch/Detectron#568]
There are 3 kinds of training configuration listed,
1x: For minibatch size 16, this schedule starts at a LR of 0.02 and is decreased by a factor of * 0.1 after 60k and 80k iterations and finally terminates at 90k iterations. This schedules results in 12.17 epochs over the 118,287 images in coco_2014_train union coco_2014_valminusminival (or equivalently, coco_2017_train).
2x: Twice as long as the 1x schedule with the LR change points scaled proportionally.
s1x ("stretched 1x"): This schedule scales the 1x schedule by roughly 1.44x, but also extends the duration of the first learning rate. With a minibatch size of 16, it reduces the LR by * 0.1 at 100k and 120k iterations, finally ending after 130k iterations.

But, with the same batch_size=16, what the difference between them? Why would someone use 2x or s1x which consumes more time if he could use 1x? Since there is no difference of GPU requirements.

And the most funny thing is, if I use 4 GPUs and follow the 1x schedule, of course I need to divide learning_rate by 2 and double the total training iterations, which will lead to learning_rate=0..01 and iteration=180000, and it is supposed to be trained successfully.

However, according to the 2x schedule, 8 GPUs with learning_rate=0.01 and iteration=180000. So, it means that using 4 GPUs could reach the performance of 8 GPUs under same configuration, does it make sense?

**Comment by ir413** [comment]
> But, with the same batch_size=16, what the difference between them? Why would someone use 2x or s1x which consumes more time if he could use 1x? Since there is no difference of GPU requirements

Please refer to the [model zoo](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md) for difference in performance between different schedules.

> And the most funny thing is, if I use 4 GPUs and follow the 1x schedule, of course I need to divide learning_rate by 2 and double the total training iterations, which will lead to learning_rate=0..01 and iteration=180000, and it is supposed to be trained successfully.

As explained in the [getting started](https://github.com/facebookresearch/Detectron/blob/master/GETTING_STARTED.md#2-multi-gpu-training) guide, equivalent schedules for different mini batch sizes can be obtained using the [linear scaling rule](https://arxiv.org/abs/1706.02677).

> However, according to the 2x schedule, 8 GPUs with learning_rate=0.01 and iteration=180000. So, it means that using 4 GPUs could reach the performance of 8 GPUs under same configuration, does it make sense?

The 2x schedule you describe here does not seem correct to me. Please refer to the [baseline configs](https://github.com/facebookresearch/Detectron/tree/master/configs/12_2017_baselines) and the description of the schedules given [here](https://github.com/facebookresearch/Detectron/blob/master/MODEL_ZOO.md#training-schedules). As mentioned above, it is possible to achieve the same performance for different mini batch sizes by using properly adjusted schedules.
